### DISCUSSION TITLE
Add Raids config setting per-world/island setting

### DISCUSSION THREAD

**Discussion by Srcsm** [BG-Software-LLC/SuperiorSkyblock2#2290] [Pending Feature Requests] [answered]
_I'm not sure this would be a bug so I've opened this request. I can open a proper issue if that's a better place._ 

Responding to a user on the discord, we noticed that raids are cancelled outside the islands world making using SSB on a server also trying to use a survival type world result in having raids being cancelled in the survival world.

![image](https://github.com/user-attachments/assets/5f62767c-8e4a-4f96-a9a3-db5a4e8086e4)
https://github.com/BG-Software-LLC/SuperiorSkyblock2/blob/94290a131fee0a962cb35c82d64ccd79f503ddd9/src/main/java/com/bgsoftware/superiorskyblock/listener/ProtectionListener.java#L629

Ideally, it would be nice if there were a config setting perhaps to list the world(s) to allow/restrict raids on, that way raids can still be allowed on a survival type world on the same server as SSB when the player isn't inside an island world. Otherwise, an added island setting to enable/disable raids maybe would help achieve the same goal?

**Comment by OmerBenGera** [comment, marked as answer]
This is a bug, it should be restricted only to island worlds.
Open a bug report regarding this, and list there anything that the plugin breaks on regular worlds.
P.S - spawn world is considered an islands world as well
